Dockerfile

```diff
@@ -37,4 +37,6 @@ COPY --from=build /go/bin/http-gateway /usr/bin
 COPY --from=build /go/bin/core-service /usr/bin
 COPY --from=build /go/bin/db-manager /usr/bin
 COPY --from=build /go/bin/dlv /usr/bin
+COPY build/jwt-public-certs /jwt-public-certs
+COPY build/test-certs /test-certs
```

Comment:
I would expect that this would be mounted at runtime but I guess it doesn't hurt to add it to the container.

Reply:
The problem is that we don't have any instructions on how to mount them at runtime (which is non-trivial using pre-built Tanka/Kubernetes deployment configurations and procedures) and our current build documentation assumes they are already available in the container.  I might be in favor of adding instructions and Tanka plumbing to mount the appropriate public key as long as it didn't add too much complexity, but until we have that, I think these keys/certs need to be added to the image to support the current procedure and documentation.

Reply:
Sounds good to me. I will make a proposal as part of the work on the infrastructure if I identify an elegant solution.